This fixes:

    test/test_cpu.py:15
      /home/maribu/Repos/software/aports/testing/py3-litex/src/litex-2023.12/test/test_cpu.py:15: DeprecationWarning: invalid escape sequence '\['
        litex_prompt = [b'\033\[[0-9;]+mlitex\033\[[0-9;]+m>']
--- a/litex-2023.12/test/test_cpu.py	2024-01-11 15:12:24.107360027 +0100
+++ b/litex-2023.12/test/test_cpu.py	2024-01-11 15:13:21.403348076 +0100
@@ -12,7 +12,7 @@ import os
 class TestCPU(unittest.TestCase):
     def boot_test(self, cpu_type, jobs, cpu_variant="standard"):
         cmd = f'litex_sim --cpu-type={cpu_type} --cpu-variant={cpu_variant} --opt-level=O0 --jobs {jobs}'
-        litex_prompt = [b'\033\[[0-9;]+mlitex\033\[[0-9;]+m>']
+        litex_prompt = [b'\033\\[[0-9;]+mlitex\033\\[[0-9;]+m>']
         is_success = True
         with open("/tmp/test_boot_log", "wb") as result_file:
             p = pexpect.spawn(cmd, timeout=None, logfile=result_file)
